tests/libcxxthrd/README.md

```diff
@@ -22,5 +22,9 @@ As tests are fixed, they should be moved from tests.broken to tests.supported.
 As tests are determined to be unsupportable, they should be moved from
 tests.broken to tests.unsupported.
 
-
+Implementation details:
+Since threads cannot be created or destroyed inside an enclave, the test implements hooks for 
+pthread_create, pthread_join and pthread_detach using the register_hooks interface 
+provided in the core for enclave use. Hence, there is additional timing or delay introduced
```

Comment:
Clarify that delay is due to transitioning to the host in these hooks.

Reply:
Fixed